dvc/path_info.py

```diff
@@ -180,16 +213,24 @@ def name(self):
 
     @property
     def parts(self):
-        return (self.scheme, self.netloc) + self._path.parts
+        return (self.scheme, self.parsed.netloc) + self._path.parts
```

Comment:
Why do you return non-normalized netloc here?

Reply:
was not sure what's expected on the client side and how it's supposed to be used, tbh

Reply:
If we are normalizing `__str__`, `.url`, `.netloc` and maybe some other, this should follow suit.